main/tests.py

```diff
@@ -1 +1,62 @@
+import pytest
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+
+
+@pytest.mark.django_db
+class TestLoginView:
+
+    def test_enter_login_page(self, client):
```

Comment:
note #61
The logic in the page should be tested, not the status code

Reply:
1. You can modify it so if the current user isn't logged in, it'll redirect from the entrypoint ("/") to the login page. *this* logic deserves a test.
2. This test doesn't really test the HTTP connectivity as it uses client, which calls Django WSGI layers directly. your tests need to check what is shown on the page itself.